client/blocks/author-selector/index.jsx

```diff
@@ -95,7 +95,7 @@ const SwitcherShell = React.createClass( {
 					{ ( this.props.fetchOptions.search || users.length > 10 ) &&
 						<Search
 							onSearch={ this._onSearch }
-							placeholder={ this.translate( 'Find Author…', { context: 'search label' } ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **27** times:
`translate( 'Find Author…', { context: 'search label'} )` *ES Score: 17*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

:information_source: This string doesn't exist elsewhere without a **context** (or with a different one). You can probably drop the context. If needed, you can use a translator comment instead to clarify meaning.